ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -977,3 +977,29 @@ def test_torch_dsplit(
         input=value[0],
         indices_or_sections=indices_or_sections,
     )
+
+
+# row_stack
+@handle_frontend_test(
+    fn_tree="torch.row_stack",
+    dtype_value_shape=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),

```

Comment:
the test seems to work fine, but you should add `num_arrays=st.integers(1, 5),` here, as currently we are only passing one array and therefore not testing the function properly :slightly_smiling_face: